src/ontology/mondo-edit.obo

```diff
@@ -130209,8 +130209,8 @@ xref: Orphanet:180176 {source="OMIM:113670", source="MONDO:equivalentTo"}
 xref: UMLS:C0405471 {source="OMIM:113670", source="MONDO:notFoundInDiseaseSubset"}
 xref: UMLS:C2676048 {source="MONDO:ncbi_mim2gene_medline", source="MONDO:notFoundInDiseaseSubset"}
 xref: UMLS:CN226754 {source="MONDO:equivalentTo"}
+is_a: MONDO:0002657 {source="https://orcid.org/0000-0001-5208-3432"} ! breast disorder
```

Comment:
was this manually added and therefore an intended change?

Reply:
yes, exactly. We obsoleted 'excess breast volume or number' and then this term was no longer in the 'breast disorder' branch, so i added it back

Reply:
Thanks, based on the convo during the curation meeting I will no longer flag when a subclass is added unless the source has a formatting issue.